src/windows/network.rs

```diff
@@ -246,4 +287,137 @@ impl NetworkExt for NetworkData {
     fn total_errors_on_transmitted(&self) -> u64 {
         self.errors_out
     }
+
+    fn mac_address(&self) -> MacAddr {
+        self.mac_addr
+    }
+
+    fn ipv4_address(&self) -> Ipv4Addr {
+        self.ipv4_addr
```

Comment:
I expect that at some point, we will add support for Ipv6 and that we shouldn't implement `Copy` for it. So that mean we should return a reference here instead (and also for `ipv4_netmask`). What do you think?

Reply:
rust has already implemented `Copy` for [Ipv6Addr](https://doc.rust-lang.org/std/net/struct.Ipv6Addr.html#impl-Copy-for-Ipv6Addr).  I guess that it is appropriate to use the value?

Reply:
Ok, nevermind me then. Didn't check if they did. Let's remain with the std then.